src/main/java/org/folio/service/inventory/InventoryBindingManager.java

```diff
@@ -0,0 +1,90 @@
+package org.folio.service.inventory;
+
+import static org.folio.service.inventory.InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS;
+import static org.folio.service.inventory.InventoryUtils.REQUESTS;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.okapi.common.GenericCompositeFuture;
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+import org.folio.rest.jaxrs.model.Piece;
+import org.folio.service.CirculationRequestsRetriever;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+public class InventoryBindingManager {
```

Comment:
Propose to rename to InventoryItemRequestManager

Reply:
Renamed